yazi-config/src/lib.rs

```diff
@@ -32,6 +33,8 @@ pub static PLUGINS: RoCell<plugins::Plugins> = RoCell::new();
 pub static PREVIEW: RoCell<preview::Preview> = RoCell::new();
 pub static TASKS: RoCell<tasks::Tasks> = RoCell::new();
 pub static THEME: RoCell<theme::Theme> = RoCell::new();
+pub static INPUTBOX: RoCell<popup::Input> = RoCell::new();
```

Comment:
Can we just `INPUT`, `SELECT` instead of `INPUTBOX`, `SELECTBOX`, since it's not a single word